### ISSUE TITLE
Switch to MMC5

### ISSUE THREAD

**Issue by dagit** [squid-man/MegaMan2Randomizer2#15]
This will give us more rom space for adding more songs and other fun tweaks.

**Comment by dagit** [comment]
I just tested this by generating a rom from version v0.5.0.29431 of the randomizer then applying an mmc5 IPS patch. The game seemed normal and playable except I noticed one show stopper and one weird thing. The show stopper was that in the refights, some of the teleporters were disabled and when I took the teleporter that would take you to quickman in the vanilla game, I got stuck in an infinite loop of fighting that boss. Every time I won and left the chamber the teleporter was still active so I would get warped back in and have to redo the fight.

The less severe thing I noticed was that I couldn't place item3 and jump off a ladder at the same time. I thought maybe it was a controller/user error at the time but then I loaded up the vanilla rom and had no issues with it. So I think it was the mmc5 patch.
